tests/libcxx/expected_results.txt

```diff
@@ -522,49 +522,49 @@ std/language.support/support.limits/limits/numeric.limits.members/traps.pass.cpp
 
 
 # *** STL BUGS ***
-# STL bug: VSO-121977 "<locale>: the enum value of std::money_base is not correct[libcxx]"
+# STL bug: GH-1112 VSO-121977 "<locale>: the enum value of std::money_base is not correct[libcxx]"
 std/localization/locale.categories/category.monetary/locale.moneypunct/money_base.pass.cpp FAIL
 
-# STL Bug: VSO-595631 <fstream> basic_filebuf doesn't comply with setbuf(0,0) requirement in the standard
+# STL bug: GH-1113 VSO-595631 <fstream> basic_filebuf doesn't comply with setbuf(0,0) requirement in the standard
 std/input.output/file.streams/fstreams/filebuf.virtuals/overflow.pass.cpp FAIL
 std/input.output/file.streams/fstreams/filebuf.virtuals/underflow.pass.cpp FAIL
 
-# STL bug: Our inheritance implementation is allowing this to compile when it shouldn't.
+# STL bug: GH-1256 Our inheritance implementation is allowing this to compile when it shouldn't.
 std/numerics/complex.number/complex.special/double_long_double_implicit.compile.fail.cpp FAIL
 std/numerics/complex.number/complex.special/float_double_implicit.compile.fail.cpp FAIL
 std/numerics/complex.number/complex.special/float_long_double_implicit.compile.fail.cpp FAIL
 
-# STL bug: regex_traits::transform() isn't following the Standard.
+# STL bug: GH-1004 regex_traits::transform() isn't following the Standard.
 std/re/re.traits/transform.pass.cpp FAIL
 
-# STL bug: Incorrect return types.
+# STL bug: GH-1260 Incorrect return types.
 std/numerics/complex.number/cmplx.over/pow.pass.cpp FAIL
 
-# STL bug: We allow fill() and swap() for array<const T, 0>.
+# STL bug: GH-942 We allow fill() and swap() for array<const T, 0>.
 std/containers/sequences/array/array.fill/fill.fail.cpp FAIL
 std/containers/sequences/array/array.swap/swap.fail.cpp FAIL
 
-# STL bug: VSO-207715 We reject array<NoDefault, 0>.
+# STL bug: GH-942 VSO-207715 We reject array<NoDefault, 0>.

```

Comment:
This doesn't seem like the same bug as line 543 - should we have separate GH bugs to track these distinct issues with `array<T, 0>`?